docs/effects/sunlight.md

```diff
@@ -0,0 +1,24 @@
+# Sunlight 
+
+Sunlight is a directional light source simulating the sun. Sun position calculations are based on [article](http://aa.quae.nl/en/reken/zonpositie.html) and inspired by [SunCalc](https://www.npmjs.com/package/suncalc). 
+
+## Constructor
+
+```js
+new Sunlight({timestamp, latitude, longitude, ...});
+```
+
+Parameters:
+* `timestamp`(Number) - unix timestamp in milliseconds.
+* `latitude`(Number) - observer's latitude in degrees.
```

Comment:
These should be pulled from the viewport at runtime, in `LightingEffect.getParameters`